packages/plugins/src/hardhat/tasks.ts

```diff
@@ -317,6 +309,11 @@ export const chugsplashProposeTask = async (
     'hardhat'
   )
 
+  const remoteExecution =
+    process.env.FORCE_REMOTE_EXECUTION === 'true'
```

Comment:
What's the context for this part?

Reply:
This is necessary to make it possible to test performing deployments against a local node with an executor running in a separate process as if the deployment was being done against a live network. This is required to test the deployment process using the UI. 

I'm using an environment variable here because I agree that this functionality should not be visible to the user. 

We should actually expand this pattern to cover all other cases where we use `const remoteExecution = (await getChainId(provider)) !== hre.config.networks.hardhat.chainId` to determine whether the execution is being performed by a remote executor. Currently, it's quite difficult to run realistic tests when I make changes to the executor b/c of this limitation.

Reply:
Ok got it, we should probably refactor this into a function `isRemoteExecution`. Want to make a ticket for that? I'll go ahead and approve this PR.